sections/futures/Trade/Trade.tsx

```diff
@@ -272,6 +275,7 @@ const Trade: React.FC<TradeProps> = () => {
 			<PositionButtons selected={leverageSide} onSelect={setLeverageSide} />
 
 			<OrderSizing
+				disabled={futuresMarketsPosition?.remainingMargin! <= zeroBN}
```

Comment:
Same here. I believe this should be:
```suggestion
				disabled={futuresMarketsPosition?.remainingMargin?.lte(zeroBN)}
```